components/camel-http/src/main/java/org/apache/camel/component/http/HttpProducer.java

```diff
@@ -45,12 +26,7 @@
 import org.apache.camel.http.common.HttpOperationFailedException;
 import org.apache.camel.http.common.HttpProtocolHeaderFilterStrategy;
 import org.apache.camel.spi.HeaderFilterStrategy;
-import org.apache.camel.support.DefaultProducer;
-import org.apache.camel.support.ExchangeHelper;
-import org.apache.camel.support.GZIPHelper;
-import org.apache.camel.support.MessageHelper;
-import org.apache.camel.support.ObjectHelper;
-import org.apache.camel.support.SynchronizationAdapter;
+import org.apache.camel.support.*;
```

Comment:
To import all packages is not preferable, did you run camel build with source check in order to pick up these issues?

Reply:
Pretty weird, it should complain. Can you please revert it?